driver.go

```diff
@@ -205,6 +213,10 @@ func (d *Driver) PreCreateCheck() error {
 		return errors.Wrap(err, "could not get location")
 	}
 
+	if d.UsePrivateNetwork == true && (d.networks == nil || len(d.networks) == 0) {
```

Comment:
Minor thing, but direct comparison with true/false is discouraged. Also `nil` slices have a defined length of 0, so `d.UsePrivateNetwork && len(d.networks) == 0` should be sufficient

Reply:
Nice, up to now no experiences with go on my side, but sounds reasonable.